binplace.targets

```diff
@@ -10,21 +10,35 @@
     <BinPlaceStuffDependsOn Condition="'$(IsReferenceAssembly)' == 'true' OR '$(IsRuntimeAndReferenceAssembly)' == 'true'">$(BinPlaceStuffDependsOn);BinPlaceReferenceAssembly</BinPlaceStuffDependsOn>
     <BinPlaceStuffDependsOn Condition="'$(IsRuntimeAssembly)' == 'true' OR '$(IsRuntimeAndReferenceAssembly)' == 'true'">$(BinPlaceStuffDependsOn);BinPlaceRuntimeAssembly</BinPlaceStuffDependsOn>
   </PropertyGroup>
+  
+  <ItemGroup>
+    <BinplaceConfiguration Include="$(BuildConfiguration)">
+      <RefPath>$(RefPath)</RefPath>
+      <RuntimePath>$(RuntimePath)</RuntimePath>
+    </BinplaceConfiguration>
+    <BinplaceConfiguration Condition="'$(IsNETCoreApp)' == 'true'" Include="netcoreapp-$(OSGroup)">
+      <RefPath Condition="'$(IsNETCoreAppRef)' == 'true'">$(BinDir)netcoreapp\pkg\ref</RefPath>
+      <RuntimePath>$(BinDir)netcoreapp\pkg\lib</RuntimePath>
+    </BinplaceConfiguration>
+    <BinplaceConfiguration Include="netstandard-$(OSGroup)">
+      <RefPath>$(RefRootPath)netstandard</RefPath>
+    </BinplaceConfiguration>
```

Comment:
We must still list it if we want a NETStandard build.  What this says is that when any project builds, if the configuration its building is the best configuration for netstandard, then put it in the netstandard ref path.